testutil-mentions/src/main/kotlin/io/spine/examples/pingh/testing/mentions/given/PredefinedGitHubSearchResponses.kt

```diff
@@ -150,4 +150,13 @@ public class PredefinedGitHubSearchResponses : GitHubSearch {
     public fun mentionsAreNotFetched() {
         areMentionsFetched = false
     }
+
+    /**
+     * Resets the instance to its initial state.
+     */
+    public fun clean() {
+        unfreeze()
```

Comment:
This is weird. Where is `freeze` in this changeset, and did the whole machine work properly before this PR?